blueman/plugins/manager/Info.py

```diff
@@ -57,6 +57,8 @@ def on_accel_activated(_group: Gtk.AccelGroup, _dialog: GObject, key: int, _modi
     dialog.set_transient_for(parent)
     dialog_content_area = dialog.get_content_area()
 
+    dialog.add_button(Gtk.STOCK_CLOSE, Gtk.ResponseType.CLOSE)
```

Comment:
```suggestion
    dialog.add_button("_Close", Gtk.ResponseType.CLOSE)
```

All the STOCK bits have been deprecated since Gtk 3.10.

Reply:
Nope, that PR does not work for me. Upstream version 2.4.6 works fine.

```
(blueman-applet:6902): dbind-WARNING **: 17:41:47.633: Couldn't connect to accessibility bus: Failed to connect to socket /run/user/0/at-spi/bus_0: Permission denied
blueman-applet 17.41.47 ERROR    PluginManager:91 load_plugin: Unable to load plugin module NMDUNSupport
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/blueman/main/NetworkManager.py", line 9, in <module>
    gi.require_version('NM', '1.0')
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/gi/__init__.py", line 122, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace NM not available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/blueman/main/PluginManager.py", line 89, in load_plugin
    importlib.import_module(self.module_path.__name__ + f".{plugin}")
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1027, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/lib/python3.13/site-packages/blueman/plugins/applet/NMDUNSupport.py", line 8, in <module>
    from blueman.main.NetworkManager import NMDUNConnection, NMConnectionError
  File "/usr/lib/python3.13/site-packages/blueman/main/NetworkManager.py", line 11, in <module>
    raise ImportError('NM python bindings not found.')
ImportError: NM python bindings not found.
blueman-applet 17.41.47 ERROR    PluginManager:91 load_plugin: Unable to load plugin module NMPANSupport
Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/blueman/main/NetworkManager.py", line 9, in <module>
    gi.require_version('NM', '1.0')
    ~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "/usr/lib/python3.13/site-packages/gi/__init__.py", line 122, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace NM not available

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.13/site-packages/blueman/main/PluginManager.py", line 89, in load_plugin
    importlib.import_module(self.module_path.__name__ + f".{plugin}")
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.13/importlib/__init__.py", line 88, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 1027, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/lib/python3.13/site-packages/blueman/plugins/applet/NMPANSupport.py", line 8, in <module>
    from blueman.main.NetworkManager import NMPANConnection, NMConnectionError
  File "/usr/lib/python3.13/site-packages/blueman/main/NetworkManager.py", line 11, in <module>
    raise ImportError('NM python bindings not found.')
ImportError: NM python bindings not found.
```

Reply:
I am not using NetworkManager on my system. My working version does not has this dependency.